### DISCUSSION TITLE
Is there a way to force a link to open in the current window or pop up a new Tauri window instead of opening it in the browser?

### DISCUSSION THREAD

**Discussion by crackleeeessyp** [tauri-apps/tauri#11809] [Q&A] [answered]
The shell plugin will open new windows in the system's default browser. How can I open a new Tauri app window or force it to open in the current window?

Electron has this feature to change new window behavior, but I couldn't find a similar feature in Tauri using Rust.

**Comment by FabianLars** [comment]
this is not directly supported yet (it's kinda possible to do it by hand by injecting js similar to the shell plugin). feel free to open a feature request though (nobody did that yet since https://github.com/tauri-apps/tauri/issues/1657#issuecomment-1031040474)

**Reply by crackleeeessyp** [reply, marked as answer]
Thanks, will create a feature request for it~
